packages/serverless/src/domain/use-case/group.ts

```diff
@@ -108,15 +113,15 @@ export class Group {
   /**
    * Retrieves detailed information about a group based on the provided parameters.
    *
-   * @param params - The parameters to identify the group, following the `TGroupParamDetail` type.
+   * @param paramGroupDetail - The parameters to identify the group, following the `TGroupParamDetail` type.
    * @param session - (Optional) The MongoDB session for transactional queries.
    * @returns A promise resolving to the group detail, conforming to the `TGroupDetail` type.
    */
   public static async detail(
-    params: TGroupParamDetail,
+    paramGroupDetail: TGroupParamDetail,
```

Comment:
You can reuse, function can have the same kine of param. It's ok and memorizable. If you define too many alias it definitely mess shit up. We don't need strong distingue here for params.

Reply:
I just refactor. remove redundant define